.github/workflows/flink-ci.yml

```diff
@@ -19,27 +19,13 @@
 
 name: "Flink CI"
 on:
-  push:
-    paths-ignore:
-    - '.github/workflows/python-ci.yml'
-    - '.github/workflows/spark-ci.yml'
-    - '.github/workflows/hive-ci.yml'
-    - '.github/workflows/cancel-duplicate-workflow-runs.yml'
-    - '.gitignore'
-    - 'dev/**'
-    - 'mr/**'
-    - 'hive3/**'
-    - 'hive3-orc-bundle/**'
-    - 'hive-runtime/**'
-    - 'spark/**'
-    - 'pig/**'
-    - 'python/**'
-    - 'python_legacy/**'
-    - 'docs/**'
-    - '.gitattributes'
-    - 'README.md'
-    - 'CONTRIBUTING.md'
   pull_request:
+    types:
+    - opened
+    - reopened
+    - ready_for_review
```

Comment:
I suspect, it's fine to omit `ready_for_review`, because it's only a change on "PR metadata" and CI should have already been triggered.

Reply:
in that case we could probably omit all of the types, since `opened` / `reopened` / `synchronize` are the defaults for `pull_request` according to https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

Reply:
Oh I thought that `synchronize` wasn't a default. Then yeah, we can omit them. Good catch!